modules/mc.py

```diff
@@ -0,0 +1,396 @@
+# mc.py a command for fetching details about a minecraft server
+
+from client import client 
+from six import string_types
```

Comment:
Why are you using `string_types` from `six`? On my systems, this only returns `(str,)`, I'm not really sure what this is achieving. I'd recommend removing this import and just using the raw `str` class instead (the code looks like that's what it expects anyways).